src/Orleans.CodeGenerator/PropertyUtility.cs

```diff
@@ -9,6 +9,8 @@ namespace Orleans.CodeGenerator
 {
     public static class PropertyUtility
     {
+        private static readonly Regex PropertyMatchRegex = new("^<([^>]+)>.*$", RegexOptions.Compiled);
```

Comment:
This isn't using the source-generated `Regex`, but I believe it should.

Reply:
> This isn't using the source-generated `Regex`, but I believe it should.

This project uses netstandard2.0 thus source generated regexps are not available there yet

Reply:
For anyone curious, it uses `netstandard2.0` because it needs to in order to operate as a C# Source Generator. I wonder if that restriction has been lifted yet.